components/Wizard/formComponents/FormGroupCustom.js

```diff
@@ -45,7 +46,9 @@ const FormGroupCustom = ({
   >
     {description && (
       <TextContent>
-        <Text component="small">{description}</Text>
+        <Text component="small">
+          <FormattedMessage id="formgroup.text" defaultMessage={description} />
+        </Text>
```

Comment:
Variables shouldn't be marked for translation since they can't be translated. Only the original strings should be marked.